.github/workflows/make-debs.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Release .deb files'
+on: push
+
+jobs:
+  build-matrix:
```

Comment:
Same. Please consider to remove -matrix.

Reply:
done.